squarelet/organizations/tests/test_querysets.py

```diff
@@ -1,5 +1,7 @@
 # Third Party
 # Django
+# Django
+# Django
```

Comment:
So the `import` formatter bugs out sometimes with the comments - usually related to removing the imports for a section but leaving the comment in (looks like there were third party imports that were removed) - the empty comment seems to mess it up - deleting the rogue section comment and re-running seems to fix it.  I will fix this one but try to check to make sure it looks reasonable.